source/codegen/common_helpers.py

```diff
@@ -85,17 +85,26 @@ def _is_custom_struct(parameter: dict) -> bool:
     return parameter["type"].startswith("struct")
 
 
-def uses_nidevice_common_message_types(functions: dict) -> bool:
-    """Whether the function has any parameters whose type is a common nidevice type."""
-    return any(
+def uses_nidevice_common_message_types(config: dict, functions: dict) -> bool:
```

Comment:
Note: This and the templates updates are general for handling nested custom types. They should come with an update to ni-fake with corresponding tests and eventually be gone in this PR.